src/App.js

```diff
@@ -1,67 +1,8 @@
 import React from "react";
+import ContructionNoticeCreate from "./ContructionNoticeCreate";
```

Comment:
There's a typo in the name, should be `ContsructionNoticeCreate` instead of `ContructionNoticeCreate`.